middle_end/flambda2/simplify/join_points.ml

```diff
@@ -24,6 +24,33 @@ type result =
     escapes : bool
   }
 
+let introduce_extra_params_in_use_env epa (env_at_use, use_id, kind) =
```

Comment:
Do you think you could make this work with `env_at_use` being a `DE.t` here ? There's already a CR at line 75 and I think now might be a good moment to address it.

Reply:
done